Tests/Data/ThermoHydroMechanics/Linear/Beam_unsealed_bimaterial/square_1e2.prj

```diff
@@ -0,0 +1,322 @@
+            <solid_specific_heat_capacity>C_s</solid_specific_heat_capacity>
+            <solid_thermal_conductivity>lambda_s</solid_thermal_conductivity>
+            <fluid_specific_heat_capacity>C_f</fluid_specific_heat_capacity>
+            <fluid_thermal_conductivity>lambda_f</fluid_thermal_conductivity>
+            <reference_temperature>T0</reference_temperature>
+            <process_variables>
+                <displacement>displacement</displacement>
+                <pressure>pressure</pressure>
+                <temperature>temperature</temperature>
+            </process_variables>
+            <secondary_variables>
+                <secondary_variable type="static" internal_name="sigma" output_name="sigma"/>
+                <secondary_variable type="static" internal_name="epsilon" output_name="epsilon"/>
+            </secondary_variables>
+            <specific_body_force>0 0</specific_body_force>
+        </process>
+    </processes>
+    <time_loop>
+        <processes>
+            <process ref="THERMO_HYDRO_MECHANICS">
+                <nonlinear_solver>basic_newton</nonlinear_solver>
+                <convergence_criterion>
+                    <type>Residual</type>
+                    <norm_type>NORM2</norm_type>
+                    <abstol>1e-5</abstol>
+                </convergence_criterion>
+                <time_discretization><type>BackwardEuler</type></time_discretization>
+                <time_stepping>
+                    <type>FixedTimeStepping</type>
+                    <t_initial>0</t_initial>
+                    <t_end>1000</t_end>
+                    <timesteps>
+                        <pair><repeat>10</repeat>
+                        <delta_t>100</delta_t></pair>
+                    </timesteps>
+                </time_stepping>
+            </process>
+        </processes>
+        <output>
+            <type>VTK</type>
+            <prefix>square_1e2</prefix>
+            <timesteps>
+                <pair><repeat>1</repeat>
+                <each_steps>10</each_steps></pair>
+            </timesteps>
+            <variables>
+                <variable>displacement</variable>
+                <variable>pressure</variable>
+                <variable>temperature</variable>
+                <variable>sigma</variable>
+                <variable>epsilon</variable>
+            </variables>
+        </output>
+    </time_loop>
+    <parameters>
+        <parameter>
+            <name>E</name>
+            <type>Group</type>
+            <group_id_property>MaterialIDs</group_id_property>
+            <index_values>
+            <index>0</index>
+            <value>20</value>
+            </index_values>
+	        <index_values>
+            <index>1</index>
+            <value>0.1</value>
+            </index_values>
+            <index_values>
+            <index>2</index>
+            <value>10</value>
```

Comment:
In the documentation the parameters for the sealed and unsealed tests are equal, but not in the project files.
Not clear, which one is correct.

Reply:
In the documentation, the parameters for the sealed and unsealed tests do not need to be equal. They do not compare with each other.

Reply:
It is fine, that the parameters are different between the sealed and unsealed cases. Which parameter values are correct? Those in the documentation, or those in the project files?

Reply:
The parameters are added in the pdf.